opm/autodiff/StandardWellV_impl.hpp

```diff
@@ -2293,9 +2293,9 @@ namespace Opm
     void
     StandardWellV<TypeTag>::
     computeWellRatesWithBhp(const Simulator& ebosSimulator,
-                            const EvalWell& bhp,
+                            const double& bhp,
                             std::vector<double>& well_flux,
```

Comment:
Any reasons for using EvalWell here?

Reply:
Probably because we can not convert a `Scalar` to `DynamiceEvaluation`?